ingestors/src/advisory/csaf.rs

```diff
@@ -0,0 +1,43 @@
+use anyhow::anyhow;
+use csaf::document::Category;
+use csaf::Csaf;
+
+use trustify_api::db::Transactional;
+use trustify_api::graph::Graph;
+
+use crate::advisory::AdvisoryMetadata;
+
+pub async fn ingest(
+    system: &Graph,
+    csaf: Csaf,
+    sha256: &str,
+    location: &str,
+) -> anyhow::Result<i32> {
+    let document_category = &csaf.document.category;
+    if !matches!(document_category, Category::Vex) {
```

Comment:
I was assuming there was a good reason since the `importer` is already not importing files that don't have the  `vex` category. So I was just doing a copy/paste to not disrupt previous assumptions/decisions. I'll be glad to remove that restriction as long as I don't remove any decision/assumption taken previously.

https://github.com/trustification/trustify/blob/4fd2dec25c977f34edad29074504146f09735785/importer/src/csaf/mod.rs#L132-L135

Reply:
Yea, that was there just to keep an initial import simple, as we don't use it so far. Or at least for RH data. But I don't think it makes sense to reject non-vex documentation at the core.

Reply:
Ok, now we are accepting all CSAF files regardless of its category